examples/euler_gravity_3d/rising_hot_sphere.py

```diff
@@ -340,7 +340,7 @@ def euler3d(kernel_language='Fortran',solver_type='classic',\
     claw.solver = solver
     claw.output_format = output_format
     claw.output_file_prefix = file_prefix
-    claw.keep_copy = False
+    claw.keep_copy = True
```

Comment:
I think it's best to leave this set to `False` by default.  If someone tries running this with a larger grid, they could easily run out of memory.